### ISSUE TITLE
maximum chromsize warning when loading pairix

### ISSUE THREAD

**Issue by dmalzl** [open2c/cooler#236]
Hi,

I am using `cooler cload pairix` to generate a cooler file from a pairix file I previously generated with `cooler csort`. I am working with an organism with a very large genome and although I increased the max_splits I keep getting "Warning: maximum chromosome size is 2^30".

I figured this is due to the pypairix library which generates the iterator for the whole chromosomes and calls a subsequent function of the pairix module `reg2bins`, which issues this warning when the iterator is created.

However, the cool file seems to be generated as expected (at least judging from the info) but I am not sure if this has any repercussion on the actual data it contains. Do you have an opinion on this?

My chromsizes are:
```
chr1p   1416415443
chr1q   1471228731
chr2p   1381227519
chr2q   1413870220
chr3p   867469519
chr3q   1525846468
chr4p   1204678091
chr4q   1215118724
chr5p   1279525745
chr5q   1285306171
chr6p   1462196291
chr6q   1625855492
chr7p   907274393
chr7q   1083033678
chr8p   745249056
chr8q   885924281
chr9p   454801983
chr9q   1001906603
chr10p  1081352037
chr10q  525520881
chr11p  305453894
chr11q  1078813816
chr12p  288882840
chr12q  857146890
chr13p  241582323
chr13q  482900735
chr14p  184541253
chr14q  436048769
```

Thanks for looking into it,
Daniel

PS: I already wrote an [issue](https://github.com/4dn-dcic/pairix/issues/66) in the pairix repo also.

**Comment by dmalzl** [comment]
I tried querying the pairix file with coordinates > 2^30 and this indeed does not return anything. In addition, I also checked the produced matrix and they are empty beyond this distance threshold. However, cload pairs solves this and the returned matrices have contacts in this region. I just previously resorted to pairix because it is a neat way to store pairs, works with juicer and also has a parameter for specifying more threads which I found convenient to have. Anyway, cload pairix is faster although just about 2x with 12 cores. 

I will just use cload pairs for now thanks for the help
